android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -1770,8 +1770,12 @@ private WritableMap createNetworkResponseInfo(final MaxNetworkResponseInfo respo
         WritableMap credentials = Arguments.createMap();
         for ( String key : credentialBundle.keySet() )
         {
-            String value = credentialBundle.getString( key, "" );
-            credentials.putString( key, value );
+            Object obj = credentialBundle.get( key );
+            if ( obj != null )
```

Comment:
curious- for what key would `obj` be null?

Reply:
Actually, I don't know.  According to the spec at https://developer.android.com/reference/android/os/BaseBundle#get(java.lang.String), it might return null.

Reply:
Only if we're using an unknown key. Was this actually an issue reported somewhere?

Reply:
I found this during testing.    Key is not a problem.   The value was boolean so that getString() failed.   If the value is always there, should we do this?
```
            credentials.putString( key, credentialBundle.get( key ).toString() );
```

Reply:
Can you print the `credentialBundle` object and send it over? I'm curious how we have booleans in credentials

Reply:
Here is the dump.   I got this while I was testing a native ad.

```
{
   "dspName":"mintegral.com",
   "waterfall":{
      "latencyMillis":231,
      "testName":"Control",
      "networkResponses":[
         {
            "error":{
               "code":-5202,
               "adLoadFailureInfo":null,
               "message":"Invalid Configuration"
            },
            "latencyMillis":37,
            "credentials":{
               "placement_id":"105320",
               "is_native":"true",
               "app_id":"2604375"
            },
            "mediatedNetwork":{
               "sdkVersion":"2.4.20220216",
               "adapterVersion":"2022.2.16.0",
               "adapterClassName":"com.applovin.mediation.adapters.LineMediationAdapter",
               "name":"LINE"
            },
            "adLoadState":2
         },
         {
            "latencyMillis":190,
            "credentials":{
               "placement_id":"native_native"
            },
            "mediatedNetwork":{
               "sdkVersion":"11.4.4",
               "adapterVersion":"11.4.4",
               "adapterClassName":"com.applovin.mediation.adapters.AppLovinMediationAdapter",
               "name":"AppLovin"
            },
            "adLoadState":1
         }
      ],
      "name":"Default Waterfall"
   },
   "adUnitId":"c946ea41e8c39bfe",
   "placement":"myplacement",
   "revenue":0.0000341096,
   "networkName":"APPLOVIN_EXCHANGE",
   "creativeId":"2828004837"
}
```
```
{
   "dspName":"twitter.com",
   "waterfall":{
      "latencyMillis":207,
      "testName":"Control",
      "networkResponses":[
         {
            "error":{
               "code":-5202,
               "adLoadFailureInfo":null,
               "message":"Invalid Configuration"
            },
            "latencyMillis":28,
            "credentials":{
               "placement_id":"105320",
               "is_native":"true",
               "app_id":"2604375"
            },
            "mediatedNetwork":{
               "sdkVersion":"2.4.20220216",
               "adapterVersion":"2022.2.16.0",
               "adapterClassName":"com.applovin.mediation.adapters.LineMediationAdapter",
               "name":"LINE"
            },
            "adLoadState":2
         },
         {
            "latencyMillis":177,
            "credentials":{
               "placement_id":"native_native"
            },
            "mediatedNetwork":{
               "sdkVersion":"11.4.4",
               "adapterVersion":"11.4.4",
               "adapterClassName":"com.applovin.mediation.adapters.AppLovinMediationAdapter",
               "name":"AppLovin"
            },
            "adLoadState":1
         }
      ],
      "name":"Default Waterfall"
   },
   "adUnitId":"c946ea41e8c39bfe",
   "placement":"myplacement",
   "revenue":0.0004968,
   "networkName":"APPLOVIN_EXCHANGE",
   "creativeId":"native_38362934_1544127260579532800"
}
```

Reply:
I'd breakpoint to see exactly which key it's returning non-string for, since all of those `"credentials"` objects look to be strings

Reply:
Sorry for the confusion.   I converted the boolean to string for the dump.   Here is the output from 'credentialBundle.toString()':

2022-07-05 15:09:16.908 13184-13184/com.applovin.enterprise.apps.demoapp I/System.out: Bundle[{app_id=2604375, is_native=true, placement_id=105320}]

So,  the dump has to be:

            "credentials":{
               "placement_id":"105320",
               "is_native":true,                      <= should have been boolean here
               "app_id":"2604375"
            },

Reply:
Makes sense